docs/site/src/demos/icons/icons.md

```diff
@@ -0,0 +1,11 @@
+# Icons
+
+[Material Icons](https://material.io/guidelines/style/icons.html) use geometric shapes to visually represent core ideas, capabilities, or topics.
+
+## Font Icons
```

Comment:
What about moving that doc section under `/style/icons`? We already have the `SvgIcon` there.